client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -189,18 +278,18 @@ export const PatternLibrary = ( {
 				searchTerm={ urlQuerySearchTerm }
 			/>
 
-			<DocumentHead title={ translate_not_yet( 'WordPress Patterns - Category' ) } />
+			<DocumentHead title={ seoContent.title } />
 
 			<PatternsHeader
-				description={ translate_not_yet(
+				description={ translate(
 					'Dive into hundreds of expertly designed, fully responsive layouts, and bring any kind of site to life, faster.'
 				) }
 				initialSearchTerm={ searchTerm }
 				key={ `${ searchFormKey }-search` }
 				onSearch={ ( query ) => {
 					setSearchTerm( query );
 				} }
-				title={ translate_not_yet( 'It’s Easier With Patterns' ) }
+				title={ translate( 'Build your site faster with patterns' ) }
```

Comment:
```suggestion
				title={ translate( 'It's Easier With Patterns' ) }
```
Was this changed based on this Slack thread? p1711383014975249-slack-C06ELHR6L9J

The existing copy has been written by the marketing team and was updated in #88622. Let's stick with that until we have some other consensus.

Reply:
Please take a look at this before merging, though, @chad1008

Reply:
Yep, that's exactly where that came from. I've committed your suggestion to leave it as it was for now, thank you for the guidance!